SnakeGame/SnakeGame/Game.cs

```diff
@@ -0,0 +1,203 @@
+﻿using System.Security.Cryptography;
+
+namespace SnakeGame
+{
+    public class Game
+    {
+        public bool SnakeIsAlive = true;
+
+        private Snake _snake = new Snake(4);
+
+        private string _user;
+        private (int, int) _applePosition;
+        private string _keyPressed;
+        private bool _hasWalls;
+        private Directions _direction = Directions.RightArrow;
+
+        public Game() { }
+
+        public void MainProcess()
+        {
+            Console.CursorVisible = false;
+
+            OpenStartingMenu();
+            // Close starting menu
+            Console.Clear();
```

Comment:
Can we clear the console in some of the functions which are called in the Main Process